app/ui-react/syndesis/src/modules/integrations/pages/create/configure/main/SaveIntegrationPage.tsx

```diff
@@ -54,16 +54,17 @@ export class SaveIntegrationPage extends React.Component {
                 history.push(resolvers.list());
               };
               const definition: IFormDefinition = {
-                description: {
-                  defaultValue: '',
-                  displayName: 'Description',
-                  type: 'textarea',
-                },
                 name: {
                   defaultValue: '',
                   displayName: 'Name',
                   required: true,
                 },
+                // tslint:disable-next-line
```

Comment:
You can add `order` to these fields to organize them how they should be, so no need to disable tslint.  Set the `name` to have `order: 0` and the description to have `order: 1`

Reply:
Oh good call, thanks! I'll probably do both things, it helps me see things the way they should be.